python/tskit/trees.py

```diff
@@ -5600,7 +5600,7 @@ def genetic_relatedness(
                 windows=windows,
                 mode=mode,
                 span_normalise=span_normalise,
-            )
+            )[:, None]
```

Comment:
Good thought, but sadly this won't work [with `windows=None`](https://tskit.readthedocs.io/en/latest/stats.html#windows). I think it *will* work, without the `[:,None]`, if you pass `[all_samples]` as the sample sets instead of `all_samples`. (passing a list of sample IDs instead of a list of lists of sample ids drops a dimension in the output; that's what's causing the problem here).

Reply:
What was the resolution here @brieuclehmann?

Reply:
@petrelharp 's suggestion of using `[all_samples]` worked! Should I only 'resolve' something once I've pushed the change?

Reply:
I think waiting for the push makes sense! (but, don't worry about pushing a lot... it's nice to know what happened with these things...)